_sass/components/_blockquote-section.scss

```diff
@@ -0,0 +1,109 @@
+$bp-quote-sm: 62.5rem;
+$bp-quote-md: 86.25rem;
+$bp-quote-lg: 106.25rem;
+$bp-quote-xl: 125rem;
+
+.c-blockquote {
+  width: 100%;
+  padding: 6rem 0;
+  margin: 4rem auto;
+  background: #F4F4F4;
```

Comment:
Is this color reused anywhere? If so, it could be put into a variable. Otherwise I think it's ok to stand alone.

Reply:
added to variables